libraries-2/src/main/java/com/baeldung/jcommander/usagebilling/cli/UsageBasedBilling.java

```diff
@@ -0,0 +1,86 @@
+package com.baeldung.jcommander.usagebilling.cli;
+
+import com.beust.jcommander.JCommander;
+import com.beust.jcommander.ParameterException;
+import com.beust.jcommander.UnixStyleUsageFormatter;
+
+public class UsageBasedBilling {
+
+    private JCommander main;
```

Comment:
`main` is a non-descript name for it. You could call it `jcommander` or `commandParser`?

Reply:
addressed